Contributing.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to Eclipse Modeling Workflow Engine
+
+Thanks for your interest in this project.
+
+## Project description
+
+Welcome to the Modeling Workflow Engine wiki. the modeling workflow engine (MWE)
```

Comment:
Typo: `wiki. the` should be `wiki. The`

Reply:
@miklossy this is the text as it is generated by tool.
i have no idea where the source is coming from.

Reply:
@cdietrich I think it is coming from the [Overview](https://projects.eclipse.org/projects/modeling.emf.mwe) page of the Eclipse MWE project.

![image](https://user-images.githubusercontent.com/20393472/74010146-1630db00-4985-11ea-9064-f2e31cbe5ba5.png)

Reply:
then please correct the text there

Reply:
@cdietrich I corrected the typo on the Eclipse MWE Wiki page:
![screenshot](https://user-images.githubusercontent.com/20393472/74519727-8eb01280-4f16-11ea-9486-113d6acc79f0.png)